apps/explorer/src/components/recent-packages-card/RecentPackagesCard.tsx

```diff
@@ -52,11 +52,11 @@ const transformTable = (events: SuiEvents) => ({
             timestamp,
             txDigest,
         }: SuiEventEnvelope): PackageTableData => {
-            if (!('publish' in event)) return {};
+            if (!(event.type === 'publish')) return {};
             return {
                 time: <TxTimeType timestamp={timestamp} />,
-                sender: <AddressLink address={event.publish.sender} />,
-                packageId: <ObjectLink objectId={event.publish.packageId} />,
+                sender: <AddressLink address={event.content.sender} />,
+                packageId: <ObjectLink objectId={event.content.packageId} />,
```

Comment:
similarly, can we add `getEventSender` and `getEventPackage` to src/types/events.ts?